src/Symfony/Component/String/Slugger/AsciiSlugger.php

```diff
@@ -88,6 +90,14 @@ public function getLocale(): string
         return $this->defaultLocale;
     }
 
+    public function enableEmoji(bool $enableEmoji = true): void
+    {
+        if (!class_exists(EmojiTransliterator::class)) {
```

Comment:
this should be `if ($enableEmoji && !class_exists(EmojiTransliterator::class))`. There is no requirement if we don't actually enable emoji support.

Reply:
Can it be also usefull to add it in « suggest » in the composer.json ?

Reply:
No, we’re not doing it anymore.

Reply:
Oki